src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.ts

```diff
@@ -135,39 +140,61 @@ export default class AuthApp extends Vue {
         this.createRecordStudioId = studioId ?? '';
     }
 
+    showError(message: string) {
+        this.showErrorDialog = true;
+        this.errorMessage = message;
+    }
+    closeErrorDialog() {
+        this.showErrorDialog = false;
+        this.errorMessage = '';
+    }
+
     async createStudio() {
-        this.showCreateStudio = false;
-        const comId = authManager.getComIdFromUrl();
-        const result = await authManager.client.createStudio({
-            displayName: this.studioName,
-            ownerStudioComId: comId,
-        });
-        await this.loadStudios();
+        try {
+            this.showCreateStudio = false;
+            const comId = authManager.getComIdFromUrl();
+            const result = await authManager.client.createStudio({
+                displayName: this.studioName,
+                ownerStudioComId: comId,
+            });
+            await this.loadStudios();
+        } catch (error) {
+            this.showError(error.message);
+        }
```

Comment:
@TroyceGowdy This function should check `result.success` just like `createRecord()` does below.